static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -163,10 +163,17 @@ define([
         }
 
         // use a #force-sw hash fragment to force service worker registration for local dev
-        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash === '#force-sw') {
+        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash.indexOf('force-sw') > -1) {
```

Comment:
Yeah I guess this is an acceptable trade-off. I'm not familiar enough with SW but, once it is registered, will it kick off at once on the next page load? (as in not wait until the `register` call in enhanced)